src/sources/docker_logs/mod.rs

```diff
@@ -1308,7 +1308,7 @@ fn line_agg_adapter(
         (stream, message, log)
     });
     let line_agg_out = LineAgg::<_, Bytes, LogEvent>::new(line_agg_in, logic);
-    line_agg_out.map(move |(_, message, mut log)| {
+    line_agg_out.map(move |(_, message, mut log, _)| {
```

Comment:
I am not sure whether it makes a difference that for lines that merged, `log` here changes from being the `LogEvent` generated by the first line, rather than the last line. I suspect not, but wanted to draw attention to this subtle change for reviewers.